src/libraries/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Browser.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Security;
+
+namespace System.IO
+{
+    public sealed partial class DriveInfo
+    {
+        public DriveType DriveType => DriveType.Unknown;
+        public string DriveFormat => "memfs";
```

Comment:
I see "memfs" is a node.js plugin. Is this the value that I would get in the browser context, where node.js is not present - would that be confusing?

Reply:
`memfs` is the default Emscripten file system that we're using to emulate an FS in the browser: https://emscripten.org/docs/api_reference/Filesystem-API.html